lib/Bio/EnsEMBL/DataCheck/Checks/CheckMultipleAlignCoverage.pm

```diff
@@ -95,7 +95,7 @@ sub tests {
 
     my $desc_2 = "genomic_align coverage matches species_tree_node_tag for mlss_id: $mlss_id";
 
-    my $summary_result = $helper->execute_single_result(-SQL => $msa_summary_sql);
+    my $summary_result = $helper->execute_single_result(-SQL => $msa_summary_sql, -NO_ERROR => 1);
```

Comment:
If this test is allowed to fail, should it be an advisory dc rather than critical?

Reply:
Adding the `-NO_ERROR` there was to make the DC fail on the `is` instead of raising an exception. The DC itself should still pass on every database.

Your comment made me go back to the DC and I have slightly refactored it the same way I have refactored the CheckLastZCoverage DC. I'm now using `is_rows_zero` and the genome_db_ids that fail are listed, e.g.
```
# Subtest: CheckMultipleAlignCoverage
    # Subtest: multi, compara, carlac_rice_epo2x_osat_ref_100
        ok 1 - genomic_align coverage matches species_tree_node_tag for mlss_id: 9910
        not ok 2 - genomic_align coverage matches species_tree_node_tag for mlss_id: 9911
        #   Failed test 'genomic_align coverage matches species_tree_node_tag for mlss_id: 9911'
        #   at /homes/muffato/workspace/src/rel101/ensembl-datacheck/lib/Bio/EnsEMBL/DataCheck/Checks/CheckMultipleAlignCoverage.pm line 98.
        #          got: '3'
        #     expected: '0'
        # Unexpected data: (n.genome_db_id, n.node_name, t.value AS tag_coverage, g.genomic_align_coverage ) = (2071, Oryza longistaminata, 185779444, 199477577)
        # Unexpected data: (n.genome_db_id, n.node_name, t.value AS tag_coverage, g.genomic_align_coverage ) = (2010, Oryza punctata strain IRGC:105690, 137413081, 152870263)
        # Unexpected data: (n.genome_db_id, n.node_name, t.value AS tag_coverage, g.genomic_align_coverage ) = (1582, Oryza brachyantha strain IRGC:101232, 90242079, 106412312)
        # Execute
        #     SELECT n.genome_db_id, n.node_name, t.value AS tag_coverage, g.genomic_align_coverage
        #       FROM species_tree_node n
        #         JOIN species_tree_root r USING(root_id)
        #         JOIN species_tree_node_tag t USING(node_id)
        #         JOIN (
        #     SELECT d.genome_db_id, SUM(ga.dnafrag_end-ga.dnafrag_start+1) AS genomic_align_coverage
        #       FROM genomic_align ga
        #         JOIN dnafrag d USING(dnafrag_id)
        #       WHERE ga.method_link_species_set_id = 9911
        #       GROUP BY d.genome_db_id
        #      )g USING(genome_db_id)
        #       WHERE n.genome_db_id IS NOT NULL
        #         AND t.tag = 'genome_coverage'
        #         AND r.method_link_species_set_id = 9911
        #         AND g.genomic_align_coverage != t.value
        #      against carlac_rice_epo2x_osat_ref_100 to replicate these results
        1..2
        # Looks like you failed 1 test of 2.
    not ok 1 - multi, compara, carlac_rice_epo2x_osat_ref_100
    #   Failed test 'multi, compara, carlac_rice_epo2x_osat_ref_100'
    #   at /homes/muffato/workspace/src/rel101/ensembl-datacheck/lib/Bio/EnsEMBL/DataCheck/DbCheck.pm line 546.
    1..1
    # Looks like you failed 1 test of 1.
not ok 1 - CheckMultipleAlignCoverage
```